docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -25,11 +25,16 @@ needs.
 
 See the :doc:`kafka-tutorial`.
 
-.. note::
+Requirements
+------------
+
+Requirements for using the connector in a catalog to connect to a Kafka
+data source are:
 
-    The minimum supported Kafka broker version is 0.10.0.
-    The connector is tested against Confluent Kafka versions 5.2.1 and 5.5.2,
-    but any intermediate or newer versions are expected to work.
+* Kafka broker version is 0.10.0 or higher, testing performed against Confluent
+  Kafka versions 5.2.1 and 5.5.2
+* Network access, by default on port 9092, from the Trino coordinator and
+  workers to the Kafka nodes.
```

Comment:
I find this sentence awkward, what about:

> *Network access from the Trino coordinator and workers to Kafka nodes

> Port 9092 is the default port and is used in the code examples.

Reply:
The sentence is the correct and what we use in all connectors. If we want to change to multiple sentences like that we should follow up with that in all of them